src/alexapi/device_platforms/dummy.py

```diff
@@ -11,36 +14,28 @@ def __init__(self, config):
 		self.should_confirm_trigger = False
 
 	def setup(self):
-		if self._pconfig['verbose']:
```

Comment:
@renekliment I do actually use normal/debug output (and will use the `logging.debug()` after rebase/rework). I just used the _verbose_ option to output additional info (the JSON request to hyperion). If we got completely rid of that option I can also include that output in the normal debug output.